schemas/layer-2.cue

```diff
@@ -1,84 +1,131 @@
 package schemas
 @go(layer2)
 
+// Layer2 describes a collection or catalog of technology-specific, threat-informed security controls
+// that can be applied to an information system.
 #Layer2: {
+    // metadata about the Layer2 collection
     metadata?: #Metadata
-
+    // one or more ControlFamily objects
     "control-families"?: [...#ControlFamily] @go(ControlFamilies)
+    // zero or more Threats that are mitigated by the controls in the ControlFamilies
     threats?: [...#Threat] @go(Threats)
+    // zero or more Capabilities in use that inform the controls in the ControlFamilies
     capabilities?: [...#Capability] @go(Capabilities)
-
+    // zero or more mapping references to controls defined in this or other Layer2 frameworks or standards
     "shared-controls"?: [...#Mapping] @go(SharedControls)
+    // zero or more mapping references to threats defined in this or other Layer2 frameworks or standards
     "shared-threats"?: [...#Mapping] @go(SharedThreats)
+    // zero or more mapping references to capabilities defined in this or other Layer2 frameworks or standards
     "shared-capabilities"?: [...#Mapping] @go(SharedCapabilities)
 }
 
-// Resuable types //
-
 #Metadata: {
+    // unique identifier for the Layer2 collection
     id: string
+    // name for the Layer2 collection
     title: string
+    // description of the Layer2 collection
     description: string
     version?: string
+    // timestamp of the last modification to the Layer2 collection
     "last-modified"?: string @go(LastModified)
+    // Applicability categories are facets that apply to the controls in this Layer2 collection. These facet values can be used to filter which controls from Layer2 are used in Layer 3 or Layer 4.
     "applicability-categories"?: [...#Category] @go(ApplicabilityCategories)
+    // List of applicable references to Layer 1 guidance, technical capabilities and threats that inform the Layer2 collection
     "mapping-references"?: [...#MappingReference] @go(MappingReferences)
 }
 
+// Category defines a facet that applies to the AssessmentRequirements in Controls. Categories typically define "levels" of compliance, for example the low, moderate and high baselines within the NIST 800-53 controls.
```

Comment:
Is there a reason we can't use "level" here? Conceptually, "category" implies a vertically-stacked set of entries on a similar theme, whereas "level" is a horizontal slice across all categories.

Using OSPS Baseline as an example, the families like "Legal" and "Documentation" are categories and levels are..."levels" 😄 

Or to use an org chart analogy, "software engineer", "project manager", and "technical marketing manager" are categories, while "junior", "senior", and "principal" are levels.

Does that make sense?

Reply:
"Facet" isn't immediately intuitive to me. For my particular consumption (OSPS Baseline), "level" makes sense, but I don't want to dictate that everyone be like us.

That said, if most/all frameworks use something that can reasonably be called levels, I think it's worth being a little restrictive here. There are hypothetically other ways to carve up the catalog, but too much flexibility sacrifices clarity. If there are notable examples of using something other than levels, I'm open to finding a broader term.

Reply:
I'm not reflexively opposed to levels as a term here.

I think most of the thoughts leading away from that term were attempting to avoid ladder-style verbiage, but... meh. Everything in this space can be sorted onto a ladder in some way. Level is fine.

Reply:
Appreciate the feedback and discussion here. I've pushed changes that try to address this naming feedback.